content/authors/collins-kirui/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Collins Kirui
+type: authors
+github: https://github.com/collinskirui223
+images:
+  - url: /engineering-education/authors/collins-kirui/avatar.jpg 
+---
```

Comment:
Please refer to this [documentation](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/UPLOAD_INSTRUCTIONS.md) and add your skillset into your bio index.md file.